### ISSUE TITLE
RegistryKeyLookupFailed for version 7.1.0-19 on Windows 10

### ISSUE THREAD

**Issue by OyewoleOyedeji** [ImageMagick/ImageMagick#4652]
### ImageMagick version

7.1.0-19

### Operating system

Windows 10

### Operating system, version and so on

10.0.18363.1556

### Description

When I try running
`$ magick .\input.png .\output.jpg`

I get an error stating the following:
```log
magick.exe: RegistryKeyLookupFailed 'CoderModulesPath' @ error/module.c/GetMagickModulePath/665.
magick.exe: no decode delegate for this image format 'PNG' @ error/constitute.c/ReadImage/572.
```

### Steps to Reproduce

1. Upgrade ImageMagick from `7.0.10-56`
2. Run `magick ./input.png ./output.jpg` (I ran this in a VSCode environment)
3. Throws the error above

### Images
![error-trace-screenshot](https://user-images.githubusercontent.com/57059138/147933280-3fa3a0cc-4f37-44d0-91c6-9254a24b0bf1.jpg)

**Comment by OyewoleOyedeji** [comment]
this issue is becoming stale

**Comment by OyewoleOyedeji** [comment]
this issue is becoming stale

**Comment by dlemstra** [comment]
Comments like that are not helpful. We have a lot of things that need our attention we will give an answer when we have time. 

How did you install ImageMagick and what kind of terminal are you using? Can you reproduce this issue from a "normal" command prompt?

**Comment by OyewoleOyedeji** [comment]
@dlemstra 
Sorry, about that. The thing is that I'm a big supporter of ImageMagick, I use it for a lot of stuff, and I really needed this to be done.

By the way you guys are very popular, like your used in Cloudmersive

**Comment by dlemstra** [comment]
Could you answer the questions I asked?

**Comment by dlemstra** [comment]
Can you test this again with the latest release? We made some changes when reading values from the registry in the latest release?

**Comment by OyewoleOyedeji** [comment]
Sure thing

**Comment by df-0** [comment]
Hello, I'm getting the same error. I installed `ImageMagick-6.9.12-77-Q16-HDRI-x64-dll.exe` on Windows 11, with headers, and Wand with `pip install Wand` on 15/02/2023 and Python 3.10.10. The keys do seem to exist in the registry (in `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\ImageMagick\Current`), so I'm not sure what the problem is.

I'm running the following code (either via VSCode or IDLE)
```
from wand.image import Image
from wand.display import display

SOURCE_IMAGE = """G:/Shared drives/RGL_102_Short_Production/blender/tests/imagemagick/SDF/singleLine.png"""


def demo():
    with Image(filename=SOURCE_IMAGE) as img:
        print(img.size)
        for r in 1, 2, 3:
            with img.clone() as i:
                i.resize(int(i.width * r * 0.25), int(i.height * r * 0.25))
                i.rotate(90 * r)
                display(i)

demo()
```

My error is 
```
Traceback (most recent call last):
  File "g:\Shared drives\RGL_102_Short_Production\blender\production\code\image\imageMagick.py", line 29, in <module>
    demo()
  File "g:\Shared drives\RGL_102_Short_Production\blender\production\code\image\imageMagick.py", line 20, in demo
    with Image(filename=SOURCE_IMAGE) as img:
  File "C:\Users\Dave\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wand\image.py", line 9365, in __init__
    self.read(filename=filename)
  File "C:\Users\Dave\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wand\image.py", line 10120, in read
    self.raise_exception()
  File "C:\Users\Dave\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\wand\resource.py", line 225, in raise_exception
    raise e
wand.exceptions.ConfigureError: RegistryKeyLookupFailed `CoderModulesPath' @ error/module.c/GetMagickModulePath/665
```


System info is 
```
OS Name	Microsoft Windows 11 Pro
Version	10.0.22621 Build 22621
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation

System Manufacturer	Acer
System Model	ConceptD CC715-72G
System Type	x64-based PC

Processor	11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz, 2304 Mhz, 8 Core(s), 16 Logical Processor(s)
BIOS Version/Date	INSYDE Corp. V1.01, 13/09/2021
SMBIOS Version	3.3
Embedded Controller Version	1.01
BIOS Mode	UEFI
BaseBoard Manufacturer	TGL
BaseBoard Product	Sparks_TLS
BaseBoard Version	V1.01
Platform Role	Mobile
Secure Boot State	On
PCR7 Configuration	Elevation Required to View
Windows Directory	C:\WINDOWS
System Directory	C:\WINDOWS\system32
Boot Device	\Device\HarddiskVolume1
Locale	Germany
Hardware Abstraction Layer	Version = "10.0.22621.819"

Time Zone	GMT Standard Time
Installed Physical Memory (RAM)	32.0 GB
Total Physical Memory	31.8 GB
Available Physical Memory	20.8 GB
Total Virtual Memory	39.3 GB
Available Virtual Memory	26.1 GB
Page File Space	7.50 GB
Page File	C:\pagefile.sys
Kernel DMA Protection	On
Virtualisation-based security	Running
Virtualisation-based security required security properties	Base Virtualisation Support
Virtualisation-based security available security properties	Base Virtualisation Support, Secure Boot, DMA Protection, UEFI Code Readonly, SMM Security Mitigations 1.0, Mode Based Execution Control, APIC Virtualisation
Virtualisation-based security services configured	
Virtualisation-based security services running	Credential Guard
Windows Defender Application Control policy	Enforced
Windows Defender Application Control user mode policy	Off
Device Encryption Support	Elevation Required to View
A hypervisor has been detected. Features required for Hyper-V will not be displayed.	
```

Hope you can help! I did a similar install on a windows 10 machine a couple of days ago and that was working fine - main difference is Python version and the fact that the working machine had a previous version of ImageMagick installed previously. I've tried reinstalling, didn't seem to make a difference, so - help would be appreciated!
